src/ast/values.rs

```diff
@@ -152,7 +156,18 @@ impl<'a> From<Value<'a>> for serde_json::Value {
     fn from(pv: Value<'a>) -> Self {
         let res = match pv {
             Value::Integer(i) => i.map(|i| serde_json::Value::Number(Number::from(i))),
-            Value::Real(d) => d.map(|d| serde_json::to_value(d).unwrap()),
+            Value::Float(f) => f.map(|f| match Number::from_f64(f as f64) {
+                Some(number) => serde_json::Value::Number(number),
```

Comment:
Is this really needed? I would imagine serde_json can deal with special float values

Reply:
`f32` or `f64` do not implement `Display`. Maybe we could use the `{:?}` here?

Reply:
We could take the JS behaviour as a reference there: 

```
JSON.stringify({ n: Infinity })
"{\"n\":null}"
JSON.stringify({ n: NaN })
"{\"n\":null}"
JSON.stringify({ n: 1.0 })
"{\"n\":1}"
```